src/webgpu/shader/validation/decl/var_access_mode.spec.ts

```diff
@@ -0,0 +1,226 @@
+export const description = `
+7.3 var Declarations
+
+The access mode always has a default value, and except for variables in the
+storage address space, must not be specified in the WGSL source. See §13.3 Address Spaces.
+`;
+
+// TODO(4128): Validate the similar rules on pointer types.
```

Comment:
I think this is why the presubmits failed. TODOs are forbidden. Stubbed `.unimplemented();` tests are encouraged.